hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: patch
+
+Improves several issues surrounding the code introduced in :ref:`version 6.131.1 <v6.131.1>`:
+
+* Improve speed of constants collection.
+* Cache constants to the ``.hypothesis`` directory for future runs.
+* Fix a ``RecursionError`` on parsing deeply nested code.
```

Comment:
```suggestion
This patch makes the new features introduced in :ref:`version 6.131.1 <v6.131.1>` much
faster, and fixes an internal error when working with deeply-nested code.
```

(I don't like dot-points very much in our changelog, because our release tend to be much more focussed that that style would usually imply)